library/lua/script-manager.lua

```diff
@@ -178,6 +178,7 @@ function get_mod_paths(installed_subdir, active_subdir)
         for _,f in ipairs(files) do
             if not f.isdir then goto continue end
             local info = get_mod_info(f.path)
+            if info == nil then goto continue end
```

Comment:
prefer `not info` over `info == nil` unless you need to disambiguate `nil` from `false`